src/tests/class_tests/openms/source/ElutionPeakDetection_test.cpp

```diff
@@ -174,6 +174,23 @@ START_SECTION((void findLocalExtrema(const MassTrace &, const Size &, std::vecto
     // SavitzkyGolay
     TEST_EQUAL(maxes.size(), 4);
     TEST_EQUAL(mins.size(), 2);
+
+    std::cout << "maxima: " << maxes[0] << ", " << maxes[1] << ", "
```

Comment:
this test writing part should be deleted for the merge?

Reply:
I can delete it